tests/unit/plugins/v1/test_nodejs.py

```diff
@@ -41,7 +41,7 @@ class Options:
             nodejs_version = nodejs._NODEJS_VERSION
             nodejs_package_manager = "npm"
             nodejs_yarn_version = ""
-            source = "."
+            source_subdir = "."
```

Comment:
We should have a test with `source_subdir` set (to an inner directory instead of `.`) and another without it, to check both codepaths. So we might define `Options` without `source_subdir` and set separate pull and build test cases for it (`@pytest.mark.parametrize` may help to run the same test method with different parameters).